### PR TITLE
Feat/mkp-details

### PR DESCRIPTION
**Pull request by iagocarvalhodev**
components of the nft details page

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,11 +20,13 @@
     "@apollo/client": "^3.3.7",
     "@sentry/react": "^5.30.0",
     "@sentry/tracing": "^5.30.0",
+    "@types/react-copy-to-clipboard": "^5.0.0",
```
Comment (fabriciomirandabr) [inline package.json:23]: This is a dev package

File: src/__stories__/components/marketplace/BuyNftButton.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/BuyNftButton.stories.tsx:1]: This is one button with different states

File: src/__stories__/components/marketplace/NftBuyShareDetails.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,14 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/NftBuyShareDetails.stories.tsx:1]: This is one button with different states

Comment (fabriciomirandabr) [reply src/__stories__/components/marketplace/NftBuyShareDetails.stories.tsx:1]: shared/buttons

File: src/__stories__/components/marketplace/BuyNftButton.stories.tsx (hunk 2 above)
Comment (fabriciomirandabr) [reply src/__stories__/components/marketplace/BuyNftButton.stories.tsx:1]: shared/buttons

File: src/__stories__/components/marketplace/NftImage.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,28 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/NftImage.stories.tsx:1]: Isolate by context  

marketplace/details

File: src/__stories__/components/marketplace/NftInfoDetails.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/NftInfoDetails.stories.tsx:1]: Isolate by context

marketplace/details

File: src/__stories__/components/marketplace/NftImage.stories.tsx (hunk 4 above)
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/NftImage.stories.tsx:1]: Isolate by context

marketplace/detail

File: src/__stories__/components/marketplace/NftInfoDetails.stories.tsx (hunk 5 above)
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/NftInfoDetails.stories.tsx:1]: Isolate by context

marketplace/detail

File: src/__stories__/components/marketplace/ShareStatsNftDetails.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,26 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/ShareStatsNftDetails.stories.tsx:1]: Isolate by context

marketplace/detail

File: src/__stories__/components/marketplace/TitleNftDetails.stories.tsx [hunk 7]
```diff
@@ -0,0 +1,19 @@
+// also exported from '@storybook/react' if you can deal with breaking changes in 6.1
```
Comment (fabriciomirandabr) [inline src/__stories__/components/marketplace/TitleNftDetails.stories.tsx:1]: Isolate by context

marketplace/detail

File: src/components/marketplace/BuyNftButton.tsx [hunk 8]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```
Comment (fabriciomirandabr) [inline src/components/marketplace/BuyNftButton.tsx:1]: This is a generic button with different states

Comment (fabriciomirandabr) [reply src/components/marketplace/BuyNftButton.tsx:1]: shared/button

different sizes

File: src/components/marketplace/NftBuyShareDetails.tsx [hunk 9]
```diff
@@ -0,0 +1,217 @@
+import { Button, Tooltip } from 'antd'
```
Comment (fabriciomirandabr) [inline src/components/marketplace/NftBuyShareDetails.tsx:1]: Isolate by context

marketplace/detail

File: src/components/marketplace/NftImage.tsx [hunk 10]
```diff
@@ -0,0 +1,55 @@
+/* eslint-disable react/prop-types */
```
Comment (fabriciomirandabr) [inline src/components/marketplace/NftImage.tsx:1]: remove

File: src/components/marketplace/NftImage.tsx [hunk 11]
```diff
@@ -0,0 +1,55 @@
...
+
+export interface NftImageProps {
+  image: string
+}
+
+export const NftImage: React.FC<NftImageProps> = ({ image }) => {
+  return (
+    <S.Content>
+      <S.Card className={image === '' ? 'bg-fail' : ''}>
+        <img src={image || notFound} alt='teste' />
```
Comment (fabriciomirandabr) [inline src/components/marketplace/NftImage.tsx:15]: Teste?

File: src/components/marketplace/NftInfoDetails.tsx [hunk 12]
```diff
@@ -0,0 +1,119 @@
+/* eslint-disable react/prop-types */
```
Comment (fabriciomirandabr) [inline src/components/marketplace/NftInfoDetails.tsx:1]: remove

File: src/components/marketplace/ShareStatsNftDetails.tsx [hunk 13]
```diff
@@ -0,0 +1,269 @@
+/* eslint-disable react/jsx-one-expression-per-line */
```
Comment (fabriciomirandabr) [inline src/components/marketplace/ShareStatsNftDetails.tsx:1]: remove

File: src/components/marketplace/ShareStatsNftDetails.tsx [hunk 14]
```diff
@@ -0,0 +1,269 @@
...
+/* eslint-disable react/prop-types */
```
Comment (fabriciomirandabr) [inline src/components/marketplace/ShareStatsNftDetails.tsx:2]: remove

**Review by fabriciomirandabr — CHANGES_REQUESTED** [review]
isolate the components from marketplace details inside 

marketplace/details  folder

File: package.json (hunk 1 above)
Comment (iagocarvalhodev) [reply package.json:24]: it has to be in dependencies if it won't give an error

**Comment by fabriciomirandabr** [comment]
@iagocarvalhodev update

**Review by fabriciomirandabr — APPROVED** [review]

**Pull request merged**